src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -235,6 +240,29 @@ protected AuthenticationResult CacheTokenResponseAndCreateAuthenticationResult(M
             }
         }
 
+        internal void CheckResponseFromBroker(MsalTokenResponse msalTokenResponse)
```

Comment:
I don't really understand the intent here... will need to have it explained in person